src/Catty/ViewController/Stage/StagePresenterSideMenuView.swift

```diff
@@ -281,6 +290,11 @@ enum SideMenuButtonType {
         button.frame.size.width = CGFloat(StagePresenterSideMenuView.buttonInitialWidthAndHeight)
         button.frame.size.height = CGFloat(StagePresenterSideMenuView.buttonInitialWidthAndHeight)
 
+        if (imageName == "stage_dialog_button_back" || imageName == "stage_dialog_button_continue" || imageName == "stage_dialog_button_restart") && UIDevice.current.orientation.isLandscape {
+            button.frame.size.width = CGFloat(StagePresenterSideMenuView.buttonInitialWidthAndHeight * 2)
```

Comment:
Checking whether the name of the image equals to some values is very dangerous: images often get renamed, there is no test for this behavior and we will forget about it, quickly running into a regression. Instead of making certain decisions based on a string comparison, please include a new method parameter. Wouldn't it be enough to forward the `type` parameter to `setupImage`?